### PR TITLE
Add 'Run a Relayer' article

### PR DESCRIPTION
**Pull request by bucanero**

### CODE REVIEW

**Comment by gagdiez** [comment]
what would be the main difference with the current relayer docs? Maybe we could just add the pagoda's relayer as an example of a relayer, or maybe examples on how to use it as an endpoint

**Comment by ravigautham** [comment]
@bucanero I think I agree with @gagdiez's comment above. Instead of this being a separate article, we should add this as a rust implementation of a relayer server on the original article. Maybe this article can somehow be integrated into the already existing "Building a Relayer" article? Or maybe we can rename that to be "Meta Transaction Relayers" to keep it more general? 

But other than that I think the content of the article is clear and solid.

**Review by anthony-near — CHANGES_REQUESTED** [review]
@bucanero everything looks good except for the omission of `/sign_meta_tx_*` endpoints and the creation of the account keys json file, which should have brackets around the json `[ ... ]`. That was my fault for not updating the readme with the appropriate instructions. README updates here: https://github.com/near/pagoda-relayer-rs/pull/89/files

File: docs/2.build/1.chain-abstraction/multichain-gas-relayer/run-relayer.md [hunk 1]
```diff
@@ -0,0 +1,155 @@
...
+
+## Basic Setup
+
+You can follow these steps to set up your local Relayer server development environment:
+
+1. [Install Rust for NEAR Development](../../../sdk/rust/intro.md)
+2. If you don't have a NEAR account, [create one](../../../1.concepts/protocol/account-model.md)
+3. With the account from step 2, create a JSON file in this directory in the format
+   ```js
+   {"account_id":"example.testnet","public_key":"ed25519:98GtfFzez3opomVpwa7i4m3nptHtc7Ha514XHMWszLtQ","private_key":"ed25519:YWuyKVQHE3rJQYRC3pRGV56o1qEtA1PnMYPDEtroc5kX4A4mWrJwF7XkzGe7JWNMABbtY4XFDBJEzgLyfPkwpzC"}
```
Comment (anthony-near) [inline docs/2.build/1.chain-abstraction/multichain-gas-relayer/run-relayer.md:58]: add `[ ... ]` https://github.com/near/pagoda-relayer-rs/pull/89/files#diff-b335630551682c19a781afebcf4d07bf978fb1f8ac04c6bf87428ed5106870f5R102

Comment (bucanero) [reply docs/2.build/1.chain-abstraction/multichain-gas-relayer/run-relayer.md]: fixed

File: docs/2.build/1.chain-abstraction/multichain-gas-relayer/run-relayer.md [hunk 2]
```diff
@@ -0,0 +1,155 @@
...
+
+## Features
+
+The open-source Rust [reference implementation of a Relayer server](https://github.com/near/pagoda-relayer-rs/) offers the following features:
+
+:::info
+Features can be combined as needed. Use of one feature does not preclude the use of any other feature unless specified.
+:::
+
+1. Cover the gas costs of end users while allowing them to maintain custody of their funds and approve transactions (`/relay`, `/send_meta_tx`, `/send_meta_tx_async`, `/send_meta_tx_nopoll`)
```
[the remark this answers is not in the document]
Comment (bucanero) [reply docs/2.build/1.chain-abstraction/multichain-gas-relayer/run-relayer.md]: fixed

**Review by anthony-near — APPROVED** [review]

**Review by ravigautham — APPROVED** [review]
Looks great!

**Pull request merged**